docs/docs/constraint-missing-not-valid.md

```diff
@@ -29,6 +29,53 @@ ALTER TABLE "accounts" ADD CONSTRAINT "positive_balance" CHECK ("balance" >= 0)
 ALTER TABLE accounts VALIDATE CONSTRAINT positive_balance;
 ```
 
+## solution for alembic and sqlalchemy
+
+Instead of:
+
+```python
+# migrations/*.py
+from alembic import op
+
+def schema_upgrades():
+    op.create_check_constraint(
+        constraint_name="positive_balance",
+        table_name="accounts",
+        condition='"balance" >= 0',
+    )
+
+def schema_downgrades():
+    op.drop_constraint(
+        constraint_name="positive_balance",
+        table_name="accounts",
+    )
+```
+
+Use:
+
+
+```python
+# migrations/*.py
+from alembic import op
+import sqlalchemy as sa
+
+def schema_upgrades():
+    op.create_check_constraint(
+        constraint_name="positive_balance",
+        table_name="accounts",
+        condition='"balance" >= 0',
+        postgresql_not_valid=True,
+    )
+    op.execute(
```

Comment:
In this case we will get an error "Using NOT VALID and VALIDATE CONSTRAINT in the same transaction will block all reads while the constraint is validated."